packages/zoe/test/unitTests/manualTimer.test.js

```diff
@@ -78,7 +77,7 @@ test('tick does not flush by default', async t => {
   const handler = Far('handler', {
     wake: scheduled => {
       woken = scheduled;
-      stallLots().then(() => (later = true));
+      void stallLots().then(() => (later = true));
```

Comment:
I think these should be `await`. `stallLots()` is just a delay. it does nothing with `void`.